components/tools/InvoiceExtractor.tsx

```
@@ -612,7 +612,7 @@ export function InvoiceExtractor() {
             </div>
 
             {/* Rate limit note */}
-            <p className="text-white/20 font-mono text-[10px] tracking-widest text-center py-2">
+            <p className="text-muted-foreground font-mono text-[10px] tracking-widest text-center py-2">
                 // 5 EXTRACTIONS PER DAY — IP-BASED LIMIT
             </p>
```

ESLint

[error] Comments inside children section of tag should be placed inside braces

(react/jsx-no-comment-textnodes)